### PR TITLE
Fix iteration caching issue

### LINKED ISSUE #6052: Some caching issue on iterations
https://www.loom.com/share/18ff06e568004fcfa35d32aa8ee185d1

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/exercism/issues/6052.

### CODE REVIEW

File: app/helpers/react_components/student/iterations_list.rb [hunk 1]
```diff
@@ -44,7 +46,8 @@ def request
                 includes(:track, :exercise, :files, :submission).
                 order(id: :desc).
                 map { |iteration| SerializeIteration.(iteration, sideload: %i[files automated_feedback]) }
-            }
+            },
+            initialDataUpdatedAt: Time.current.to_i
```
Comment (iHiD) [inline app/helpers/react_components/student/iterations_list.rb]: Could this be camel_case pls.

**Comment by iHiD** [comment]
Thanks for sorting this. Could you fix the camel_case thing then I'll merge :)

**Comment by kntsoriano** [comment]
Done.

**Comment by iHiD** [comment]
Great. Thanks man

**Pull request merged**